statistics/handle/handle.go

```diff
@@ -159,6 +165,7 @@ func (h *Handle) execRestrictedSQLWithSnapshot(ctx context.Context, sql string,
 
 // Clear the statsCache, only for test.
 func (h *Handle) Clear() {
+	// TODO: Here h.mu seems to protect all the fields of Handle. Is is reasonable?
```

Comment:
After this pr, the `h.mu` doesn't protect all the fields of the Handle, right?

Reply:
IMHO, `h.mu` never protects all the fields of the Handle. It is only responsible for the fields in its own struct(`ctx`, `rateMap`, `pid2tid` and `schemaVersion`). `h.statsCache` has its own mutex and is not protected by `h.mu`.